pkgs/development/python-modules/dictances/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage
+, fetchPypi
+, isPy3k
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "dictances";
+  version = "1.5.1";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1xc89j6n5j9i8jk6qv2y87wcny3aq5g06w26b91z1ycm8mkyayzy";
+  };
+
+  # Missing some nixpkgs
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Distances and divergences between distributions implemented in python";
+    homepage = "https://github.com/LucaCappelletti94/distances";
```

Comment:
Is the canonical homepage `dictances` or `distances`? `distances` seems to redirect to `dictances` so that might suggest this is wrong.

Reply:
Ha! Good catch :)

I got really confused by this naming...